src/frontend/reducers.js

```diff
@@ -15,4 +15,4 @@ export default req.keys().reduce((acc, key) => {
   acc[name] = req(key);
 
   return acc;
-}, { router: routerStateReducer, form: formReducer });
+}, { routing: routeReducer, form: formReducer });
```

Comment:
I was thinking about too... do you prefer router key. I saw `routing` in the docs and ended up using it, but I guess nothing prevents from using `router`.